Automattic/jetpack
# [Carousel]: Handle error if swiper fails to load

Fixes #20235

#### Changes proposed in this Pull Request:

* Currently if Swiper JS fails to load when a user clicks a gallery image to open the carousel, the loading spinner will spin indefinitely, locking up the page. This PR detects the error and removes the loading spinner so that the user can continue using the rest of the page.

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Modify the jetpackSwiperLibraryPath.url global in your browser console to point to an invalid URL, to simulate an error when loading the carousel JS
* Click on an image, to open the carousel
* The loading spinner should appear for a moment and then disappear. You should be able to continue using the page.
